drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml

```diff
@@ -1,16 +0,0 @@
-name: matter-motion-battery-illuminance
```

Comment:
Can we safely delete this? How did you confirm it is not being used? Same for the profile below

Reply:
If there is a superset query, you could link it here for posterity :)

Reply:
See superset query that is mentioned below

Reply:
That link just goes to the supserset website but not actually the query you used. You'll have to specifically share the query, which you should be able to find next to the `Run` button

Reply:
that should be updated now, sorry about that!